app/helpers/teachers_helper.rb

```diff
@@ -0,0 +1,2 @@
+module TeachersHelper
```

Comment:
We can probably remove this file too. We can always add it back when we actually use it.